frontend/src/components/data-outputs/data-output-card/data-output-card.component.tsx

```diff
@@ -0,0 +1,214 @@
+                    type: 'success',
+                });
+            } catch (_error) {
+                dispatchMessage({
+                    content: t('Failed to unlink dataset'),
+                    type: 'error',
+                });
+            }
+        },
+        [unlinkDataset, dataOutput.id, t],
+    );
+
+    const handleDragStart = (event: React.DragEvent) => {
+        event.dataTransfer.setData(
+            'text/plain',
+            JSON.stringify({
+                type: 'data-output',
+                id: dataOutput.id,
+                name: dataOutput.name,
+            }),
+        );
+
+        event.dataTransfer.effectAllowed = 'copyMove';
+
+        // Add dragging class to the card
+        const cardElement = event.currentTarget as HTMLElement;
+        cardElement.classList.add(styles.dragging);
+
+        onDragStart?.();
+    };
+
+    const handleDragEnd = (event: React.DragEvent) => {
+        const cardElement = event.currentTarget as HTMLElement;
+        cardElement.classList.remove(styles.dragging);
+        onDragEnd?.();
+    };
+
+    const canRemove = deleteAccess?.allowed ?? false;
+
+    const getDeleteDescription = () => {
+        if (dataOutput.dataset_links && dataOutput.dataset_links.length > 0) {
+            return (
+                <Flex vertical>
+                    {t(
+                        'Are you sure you want to delete this technical asset? This asset is still used in the following output ports:',
+                    )}
+                    <List
+                        size="small"
+                        dataSource={dataOutput.dataset_links}
+                        renderItem={(link) => (
+                            <List.Item style={{ padding: '0 0', border: 'none' }}>• {link.dataset.name}</List.Item>
+                        )}
+                    />
+                </Flex>
+            );
+        }
+        return t(
+            'Are you sure you want to delete this technical asset? This can have impact on downstream dependencies',
+        );
+    };
+
+    return (
+        <Card className={styles.card} draggable onDragStart={handleDragStart} onDragEnd={handleDragEnd}>
+            <Flex className={styles.cardContent}>
+                <Flex className={styles.dragHandle}>
+                    <HolderOutlined />
+                </Flex>
+                <Flex vertical className={styles.mainContent}>
+                    <Flex justify="space-between" align="flex-start">
+                        <Flex gap={12} align="center">
+                            <CustomSvgIconLoader
+                                iconComponent={getDataOutputIcon(dataOutput.configuration.configuration_type)}
+                            />
+                            <Flex>
+                                <Link to={createDataOutputIdPath(dataOutput.id, dataOutput.owner_id)}>
+                                    <Typography.Title level={5} className={styles.title}>
```

Comment:
styles.title css class sets font size, this clashes with setting the level on Title, same with font-weight, you can set the property bold on Title and this is preferedd.

I also notice other css properties for link (a), but there is no link here?